app/services/exports/case_log_export_service.rb

```diff
@@ -45,13 +45,27 @@ def retrieve_case_logs
 
     def build_export_string_io(case_logs)
       doc = Nokogiri::XML("<forms/>")
+      pattern_age = /age\d_known/
+      omitted_attrs = %w[ethnic_group]
 
       case_logs.each do |case_log|
         form = doc.create_element("form")
         doc.at("forms") << form
         case_log.attributes.each do |key, _|
-          value = case_log.read_attribute_before_type_cast(key)
-          form << doc.create_element(key, value)
+          # omit _known fileds from export
+          if key.starts_with?("details_known_") || pattern_age.match(key) || omitted_attrs.include?(key)
+            next
+          else
+            # combine ethnic fields
+            # rubocop:disable Style/ConditionalAssignment
+            if key == "ethnic"
+              value = case_log.read_attribute_before_type_cast("ethnic") || case_log.read_attribute_before_type_cast("ethnic_group")
```

Comment:
I think we should be combining these in the model (like we do for other derived variables), not just in the export?

Reply:
sorry only just saw this. So via a `get_` method?

Reply:
Something like:

`self.ethnic = ethnic || ethnic_group`

In the before validation. (Assuming `ethnic` is the fieldname that should be exported).

Reply:
oh OK so in this method:

`def set_derived_fields!`

Reply:
Yep